neo/io/nestio.py

```diff
@@ -482,8 +500,8 @@ def read_segment(self, gid_list=None, time_unit=pq.ms, t_start=None,
         gid_list : list, default: None
             A list of GDF IDs of which to return SpikeTrain(s). gid_list must
             be specified if the GDF file contains neuron IDs, the default None
-            then raises an error. Specify an empty list [] to retrieve the spike
-            trains of all neurons.
+            then raises an error. Specify an empty list [] to retrieve the
+            spike trains of all neurons.
         time_unit : Quantity (time), optional, default: quantities.ms
             The time unit of recorded time stamps in DAT as well as GDF files.
         t_start : Quantity (time), optional, default: 0 * pq.ms
```

Comment:
This is not quite true, the default is None. The identical discrepancy is found in a few doc strings.